src/meta/src/rpc/ddl_controller.rs

```diff
@@ -1360,8 +1361,12 @@ impl DdlController {
             .get_upstream_root_fragments(fragment_graph.dependent_table_ids())
             .await?;
 
-        let upstream_actors: HashMap<_, _> = upstream_root_fragments
+        // XXX: do we need to filter here?
```

Comment:
Nope, IIUC we don't need this filter. It could be shared source streaming job. We'd better change the naming in `CreateStreamingJobContext` and `Command::CreateStreamingJob` accordingly.